tests/integration/optimizers/test_optimizer.py

```diff
@@ -60,18 +60,20 @@ def test_optimizer_single_flow(tmpdir, config):
 
 def test_optimizer_multi_flow(tmpdir, config):
     multi_flow_runner = MultiFlowRunner(
-        SingleFlowRunner(
-            flow_yaml='tests/integration/optimizers/flow1.yml',
-            documents=document_generator(10),
-            request_size=1,
-            execution_method='search',
-        ),
-        SingleFlowRunner(
-            flow_yaml='tests/integration/optimizers/flow2.yml',
-            documents=document_generator(10),
-            request_size=1,
-            execution_method='search',
-        )
+        [
+            SingleFlowRunner(
+                flow_yaml='tests/integration/optimizers/flow.yml',
```

Comment:
why not making it independent from where the tests is run. this way is easier to move tests around eventually

Reply:
That is a legacy from me in the optimizer integration tests. Shame on me. I'll fix it.